lib/cinematheque/netflix.rb

```diff
@@ -1,5 +1,18 @@
-# class Netflix < Cinematheque::MovieCollection
-#   def show
-#     @collection
-#   end
-# end
+# Online theatre
+class Netflix < Cinematheque::MovieCollection
+  def show(**fields)
+    fields.reduce(all) do |acc, (key, value)|
+      return raise 'Wrong argument' unless acc[0].respond_to?(key)
+
+      filter(acc, key, value)
+    end
+  end
+
+  private
+
+  def filter(acc, key, value)
+    acc.filter do |el|
```

Comment:
I think it is better to move `filter` enum call to `show` method, and create smth like `apply_filter` function intead this one, where you can particularily check the key and raise accordingly.